lib/SQL/Translator/Schema/Constraint.pm

```diff
@@ -56,6 +56,7 @@ Object constructor.
       match_type       => 'full',        # how to match
       on_delete        => 'cascade',     # what to do on deletes
       on_update        => '',            # what to do on updates
+      deferrable       => 0,             # whether to set DEFERRABLE, if supported by the database
```

Comment:
@Altreus Ah I now understand. I thought you implemented the deferrable flag as an `{extra}` since it is supported by one and onky one highly non-standard engine. Hence why I was talking about `sqlt_extra` and was confused by how `names` etc applies in the other PR.

I **really** recommend moving this to `extra` like all other non-standard extension, e.g.: https://metacpan.org/pod/SQL::Translator::Producer::MySQL#Extra-attributes

Then the entire not on what is and isn't ignored in https://github.com/dbsrgits/dbix-class/pull/88/files#diff-14a995427a39511927479ae4ffbbb9c7R768 becomes moot.

@ilmari thoughts?

Reply:
deferrable constraints are part of the SQL standard, so I'm with @ilmari's original implementation here (sorry to revive a 6-year old thread, i'm just reviewing this to merge b/c I need it for $work)